### DISCUSSION TITLE
Description of frePPLe objects and attributes | sample Python scripts

### DISCUSSION THREAD

**Discussion by makarov-m** [frePPLe/frepple#428] [Q&A] [answered]
Hi Everyone!

currently, I'm trying to import data from files (csv or excel) to frepple. How I can access frepple objects and what attributes do they have? Is there any description?
I tried to read and see some examples here [https://frepple.org/docs/6.18/integration-guide/low-level-integration/python.php?highlight=memory](https://frepple.org/docs/6.18/integration-guide/low-level-integration/python.php?highlight=memory), however, a more detailed description is located at the links in the bottom corner, but unfortunately, these links don't work.

Thanks, Maxim.

**Comment by jdetaeye** [comment, marked as answer]
There is no doc of that capability, just an very good examples. 

The correct links for the examples are https://github.com/frePPLe/frepple/blob/master/freppledb/input/commands/load.py and https://github.com/frePPLe/frepple/blob/master/freppledb/output/commands.py

Your boss can provide you another example from another customer.
